inference-engine/src/inference_engine/ie_plugin_cpp.hpp

```diff
@@ -88,6 +88,10 @@ class InferencePlugin {
         PLUGIN_CALL_STATEMENT(return ExecutableNetwork(actual->LoadNetwork(network, config, context), actual));
     }
 
+    ExecutableNetwork LoadNetworkFromFile(const std::string& modelPath, const std::map<std::string, std::string>& config) {
```

Comment:
can we have the same name `LoadNetwork`? Just to denote that the things are the same and it does not correlate with Export / Import